make-file-list.sh

```diff
@@ -13,7 +13,7 @@ SH_FOR_REBASE=dash
 PACKAGE_EXCLUDES="db info heimdal git util-linux curl git-for-windows-keyring
 	mingw-w64-p11-kit"
 EXTRA_FILE_EXCLUDES=
-UTIL_PACKAGES="sed awk grep findutils coreutils"
+UTIL_PACKAGES="sed awk grep findutils coreutils bash-completion"
```

Comment:
Since `bash-completion` is firmly in the "interactive usage" space, it is incorrect to include it even in MinGit.